x-pack/plugins/screenshotting/common/errors.ts

```diff
@@ -13,3 +13,5 @@ export class FailedToSpawnBrowserError extends Error {}
 export class BrowserClosedUnexpectedly extends Error {}
 
 export class FailedToCaptureScreenshot extends Error {}
+
+export class InsufficientMemoryAvailableOnCloudError extends Error {}
```

Comment:
The error code should also be explicitly referenced in the `errorCodesSchema*` objects in the telemetry schema file: `x-pack/plugins/reporting/server/usage/schema.ts`